versions/3.0.md

```diff
@@ -2214,8 +2215,8 @@ definitions:
 A simple object to allow referencing other definitions in the specification.
 It can be used to reference parameters and responses that are defined at the top level for reuse.
 
-The Reference Object is a [JSON Reference](http://tools.ietf.org/html/draft-pbryan-zyp-json-ref-02) that uses a [JSON Pointer](http://tools.ietf.org/html/rfc6901) as its value.
-For this specification, only [canonical dereferencing](http://json-schema.org/latest/json-schema-core.html#anchor27) is supported.
+The Reference Object is a [JSON Reference](https://tools.ietf.org/html/draft-pbryan-zyp-json-ref-03) that uses a [JSON Pointer](http://tools.ietf.org/html/rfc6901) as its value.
```

Comment:
I'm not sure I understand what's being said here.  Is the _Reference Object_ **only** for local references, as in references within the same document?  If that's the case, the value is itself a _JSON Pointer_ as a _JSON Reference_ allows both local and remote references.